tests/sync/test_git.py

```diff
@@ -112,7 +112,7 @@ def test_repo_git_obtain_full(
 
 @pytest.mark.parametrize(
     # Postpone evaluation of options so fixture variables can interpolate
-    "constructor,lazy_constructor_options",
+    "constructor,lazy_constructor_options,untracked_files",
```

Comment:
I suppose we could rename to `has_untracked_files`

Reply:
I agree.